src/operator/databaseconfigurator/postgres.go

```diff
@@ -0,0 +1,528 @@
+package databaseconfigurator
```

Comment:
This package will be better located under `src/operator/controller/intents_reconcilers/database`

Reply:
Did not review this file because I guess it was copied almost as-is from otterize/cloud. I will say it needs some love and attention, but leaving that out of scope for now.